src/serializer/ResidualHeapInspector.js

```diff
@@ -132,15 +131,6 @@ export class ResidualHeapInspector {
         // length property has the correct descriptor values
         return true;
       }
-    } else if (isReactElement(val)) {
-      // we don't want to visit/serialize $$typeof, _owner and _store properties
```

Comment:
Where is this logic now? Was it unnecessary?

Reply:
Now we have a dedicated visitor, we simply don't visit these at all. The above code was all very hacky really. I'm happy to see this hacky code gone now!